### ISSUE TITLE
Support for PHP 8.3 for apigee-client-php

### ISSUE THREAD

**Issue by divya-intelli** [apigee/apigee-client-php#352]
**Is your feature request related to a problem? Please describe.**
Add support for PHP 8.3 in Github action.

**Comment by sebastianluczak** [comment]
Would it be possible to add support for Symfony 7 as well in scope of this work?

Installation on PHP 8.3 under Symfony 7 ends with:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - apigee/apigee-client-php[2.0.0, ..., 2.0.10] require php ^7.1 -> your php version (8.3.2) does not satisfy that requirement.
    - apigee/apigee-client-php[2.0.11, ..., 2.0.13] require php ^7.3 -> your php version (8.3.2) does not satisfy that requirement.
    - apigee/apigee-client-php[2.0.14, ..., 2.0.19] require symfony/options-resolver ^4.0 -> found symfony/options-resolver[v4.0.0, ..., v4.4.44] but the package is fixed to v7.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - apigee/apigee-client-php[2.1.0, ..., 2.1.3] require php ~8.0.0 || ~8.1.0 -> your php version (8.3.2) does not satisfy that requirement.
    - apigee/apigee-client-php[3.0.0, ..., 3.0.1] require symfony/options-resolver ^5.4 -> found symfony/options-resolver[v5.4.0, ..., v5.4.21] but the package is fixed to v7.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - apigee/apigee-client-php[3.0.2, ..., 3.0.4] require symfony/options-resolver ^6.3 -> found symfony/options-resolver[v6.3.0, v6.4.0] but the package is fixed to v7.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires apigee/apigee-client-php * -> satisfiable by apigee/apigee-client-php[2.0.0, ..., 2.1.3, 3.0.0, ..., 3.0.4].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require apigee/apigee-client-php:*" to figure out if any version is installable, or "composer require apigee/apigee-client-php:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by divya-intelli** [comment]
@sebastianluczak ,
As of now we will be prioritizing PHP 8.3. 
Have created a separate ticket for symfony 7 #354  which will be prioritized later.
